apps/docs/content/guides/auth/social-login/auth-google.mdx

```diff
@@ -341,9 +341,9 @@ Most web apps and websites can utilize Google's [personalized sign-in buttons](h
    // Use 'nonce' when invoking the supabase.auth.signInWithIdToken() method
    ```
 
-### One-tap with NextJS
```

Comment:
[This page](https://docs-git-chore-update-to-next-15-supabase.vercel.app/docs/guides/auth/social-login/auth-google?queryGroups=environment&environment=server&queryGroups=framework&framework=nextjs) has another `createClient` which needs await.